packages/backend-defaults/report-scheduler.api.md

```diff
@@ -16,6 +18,8 @@ export class DefaultSchedulerService {
     database: DatabaseService;
     logger: LoggerService;
     rootLifecycle?: RootLifecycleService;
+    httpRouter?: HttpRouterService;
+    pluginMetadata?: PluginMetadataService;
```

Comment:
eh, tbh I'm kinda thinking it's worth adding these as required dependencies straight away, unless you view this as a useful feature flag to disable the behavior by overriding the factory?

Reply:
Yeah I agree. Made all arguments required while I was at it, and updated the changeset wording accordingly.